internal/misc/http/helpers.go

```diff
@@ -21,41 +22,41 @@ func GetBaseURL(r *http.Request) string {
 	if url := os.Getenv("BASE_URL"); url != "" {
 		return url
 	}
-	return fmt.Sprintf("%s://%s/%s", r.URL.Scheme, r.Host, r.URL.Path)
+	return fmt.Sprintf("%s://%s/", r.URL.Scheme, r.Host)
 }
 
-func WriteResponse(w http.ResponseWriter, r *http.Response) {
-	for k, vs := range r.Header {
+func UnmarshalByteJson(data io.Reader, v any) error {
+	json_data, err := ioutil.ReadAll(data)
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(json_data, v)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func WriteHeaders(w http.ResponseWriter, headers http.Header) {
+	for k, vs := range headers {
 		for _, v := range vs {
 			w.Header().Set(k, v)
 		}
 	}
```

Comment:
i think that even if it get's duplicated in different packages, this `func WriteHeaders` can become a private method in any `API` and we keep it in the handlers file